packages/@uppy/tus/src/index.js

```diff
@@ -464,6 +466,7 @@ module.exports = class Tus extends Plugin {
         this.uppy.emit('upload-success', file, uploadResp)
         this.resetUploaderReferences(file.id)
         queuedRequest.done()
```

Comment:
I think these should also be flipped. This is when a 'remote' upload has finished, eg. when you select files from Google Drive.